serverless.yml

```diff
@@ -0,0 +1,114 @@
+# Welcome to Serverless!
```

Comment:
@mbast100 I've kept all the commented out config in this file so that we can see what we can do with `serverless`.

We'll clean it up after we figure out what we need.